debian/changelog

```diff
@@ -1,3 +1,9 @@
+opencast-pyca (3.2-1) unstable; urgency=medium
+
+  * New upstream release
+
+ -- Lars Kiesow <lkiesow@uos.de> Sat, 18 Jul 2020 12:26:53 AM CEST
+
```

Comment:
Seems like Debian is not really happy with your new `debian/changelog` entry, see https://app.circleci.com/pipelines/github/elan-ev/pyca-debian/64/workflows/9a3095d8-435d-4d11-aea1-15c6ff53ffcb/jobs/89/parallel-runs/0/steps/0-104

The file format is very fragile and its best to not change it manually but use the Debian Devscripts, which seems to be also available for Fedora, see https://fedora.pkgs.org/31/fedora-x86_64/devscripts-2.19.6-2.fc31.x86_64.rpm.html

There is a script called `dch` or `debchange`. So for example to create a this entry you could execute

```bash
DEBFULLNAME="Lars Kiesow" DEBEMAIL="lkiesow@uos.de" dch -v 3.2-1 -D unstable "New upstream Release"
```